src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -98,6 +98,14 @@
                             {{ $t('draw_button_delete_last_point') }}
                         </button>
                     </div>
+                    <!-- v-html could lead to some vulnerability if the html
+                    given is not thrustworthy. This makes the google doc a
+                    sensitive document (even more so than before) since an
+                    attacker which gained access to it could change the html
+                    here to cause damages to all users. Just leaving a comment
+                    here to remind us of that.-->
```

Comment:
Technically the attacker would also need to have write access to this repo, and an AWS credentials that can write on our S3 buckets.
At this point, one would not just alter some obscure translation key if he has nefarious intent 😉